grants/search.js

```diff
@@ -812,6 +812,22 @@ async function fetchGrants(mongo, locale = 'en', queryParams) {
      */
     const totalGrants = await mongo.grantsCollection.find({}).count();
 
+    const getSingleGrant = async (showNewFirst = false) => {
```

Comment:
The boolean logic feels a tad awkward. Also not sure if the `.project` does more work than needed? Maybe keeping the functions separate?

```suggestion
    // Using get from lodash/fp?
    const getDate = get('[0].awardDate');
    const firstGrant = await mongo.grantsCollection.find({}).sort({
        awardDate: -1 
    }).limit(1).toArray();

    const lastGrant = await mongo.grantsCollection.find({}).sort({
        awardDate: 1
    }).limit(1).toArray();
    
    const grantDates = {
        end: getDate(firstGrant)
        start: getDate(lastGrant)
    };
```